src/index.js

```diff
@@ -1,4 +1,4 @@
 import { MicroscopyViewerLayer } from './layers';
 import { MicroscopyViewer } from './microscopy-viewer';
 
-export { MicroscopyViewerLayer, MicroscopyViewer };
+export { MicroscopyViewerLayer as ViewerLayer, MicroscopyViewer as Viewer };
```

Comment:
Not here, but I think we need to change `MicroscopyViewerLayerBase` as well for consistency.

Reply:
Yes: I just wanted to make a PR that had the least chance of causing any merge headaches for now, and I'll be happy to do a thorough renaming afterwards. (but if you think consistency now is more important than getting something out quickly, that's reasonable, too.)